tests/shared/src/test/scala-2/zio/schema/DynamicValueSpec.scala

```diff
@@ -8,6 +8,18 @@ import zio.test.{ Sized, TestConfig, _ }
 
 object DynamicValueSpec extends ZIOSpecDefault {
 
+  case class Person(name: String, age: Int)
```

Comment:
You should not need any classes at all, because you are testing `DynamicValue`. You should create all DynamicValue manually.

Reply:
Thanks! Resolved in the next revision.